cognite/client/_api_client.py

```diff
@@ -52,19 +52,23 @@ def _init_requests_session():
     session.mount("https://", adapter)
     session_with_retry.mount("http://", adapter_with_retry)
     session_with_retry.mount("https://", adapter_with_retry)
-    return session, session_with_retry
 
+    if not verify_ssl:
+        import urllib3
+
+        urllib3.disable_warnings()
+        session.verify = False
+        session_with_retry.verify = False
 
-_REQUESTS_SESSION, _REQUESTS_SESSION_WITH_RETRY = _init_requests_session()
+    return session, session_with_retry
 
 
 class APIClient:
     _RESOURCE_PATH = None
     _LIST_CLASS = None
 
     def __init__(self, config: utils._client_config.ClientConfig, api_version: str = None, cognite_client=None):
-        self._request_session = _REQUESTS_SESSION
-        self._request_session_with_retry = _REQUESTS_SESSION_WITH_RETRY
+        (self._request_session, self._request_session_with_retry) = _init_requests_session(config.verify_ssl)
```

Comment:
This changes the behavior a bit, instead of having two global requests sessions each client will now have its own two request sessions. Unless there's a reason to, we probably shouldn't change this.

Reply:
I did it this way because the _init_requests_session() now takes the argument verify_ssl which is not known before the constructor of APIClient is called.

Reply:
@NilsBarlaug-cognite / @erlendvollset: could you think of any reason why having two sessions per client, instead of two sessions globally, might be an issue?